brage-migration-common/src/main/java/no/sikt/nva/brage/migration/common/model/record/FundingSource.java

```diff
@@ -0,0 +1,28 @@
+package no.sikt.nva.brage.migration.common.model.record;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import java.util.Map;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+public class FundingSource {
```

Comment:
Why are the models here classes instead of records? Could they be records?

Reply:
This project is written in java 11 :)